packages/remirror__theme/src/extension-code-block-theme.ts

```diff
@@ -7,6 +7,22 @@ export const WRAP = css`
   white-space: pre-wrap !important;
 `;
 
+export const LANGUAGE_SELECT = css`
+  pre[class*='language-'] {
+    padding: 2em 1em !important;
```

Comment:
Why do we need `important` here? Is there a better way, like increasing the selector level?